pkg/api/api.go

```diff
@@ -435,5 +435,8 @@ func (hs *HTTPServer) registerRoutes() {
 	r.Get("/api/snapshots-delete/:deleteKey", reqSnapshotPublicModeOrSignedIn, Wrap(DeleteDashboardSnapshotByDeleteKey))
 	r.Delete("/api/snapshots/:key", reqEditorRole, Wrap(DeleteDashboardSnapshot))
 
+	// Health check
+	r.Get("/api/health", hs.healthHandler)
```

Comment:
If we add a route for this we can remove the handler as a middleware. https://github.com/grafana/grafana/blob/master/pkg/api/http_server.go#L327 Currently the middleware will catch the requests.

Reply:
@bergquist You mean I can just remove https://github.com/grafana/grafana/blob/master/pkg/api/http_server.go#L327?

Reply:
@bergquist Making the assumption that's what you mean, and pushed a commit to remove it :) Please let me know in case I misunderstood.

Reply:
:+1: